### DISCUSSION TITLE
Request Materials

### DISCUSSION THREAD

**Discussion by siwaluh** [siwaluh/TemplateMaterials#1] [Ideas]
Tempat Request Materials

**Comment by magicmanhwa** [comment]
Source code Anime info Komikindo sama multitab anime info min

**Reply by siwaluh** [reply]
Done.

**Reply by magicmanhwa** [reply]
Terima kasih admin, sukses terus untuk kedepannya, ditunggu project2 hebat selanjutnya

**Comment by bringbackold** [comment]
Min, bisa buatkan html css seperti ini? (yang server sama episode) Kalo nggak beritahu saja ini namanya apa, biar aku search di google buat cari 
![Screenshot_8](https://github.com/siwaluh/TemplateMaterials/assets/149891096/82ffe5ae-e791-47bf-85e7-ea1e7adb06dc)

**Reply by siwaluh** [reply]
Url web nya apa?

**Reply by bringbackold** [reply]
https://yozakuraz.github.io/index.html

**Reply by bringbackold** [reply]
sekalian min, itu kok dia bisa buat web streaming pake github y

**Reply by siwaluh** [reply]
Semua type web bisa di buat lewat [GITHUB PAGES](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages), Selama tidak melanggar.

**Reply by bringbackold** [reply]
up

**Reply by siwaluh** [reply]
Done, meski agak lambat.

**Reply by bringbackold** [reply]
makasih banyak min, itu aku minta buatin opsi pilih server aja sama episodenya, soalnya udah ada streamingnya, gimana ya kalo mau ambil itunya aja, soalnya aku males jika harus ubah streaming (ubah satu2 htmlnya) maaf banget sebelumnya min

**Reply by animeblogs** [reply]
kode html untuk nambah servernya gimana min?

**Reply by siwaluh** [reply]
html streaming yang kam punya gimana?
dilihat dulu. baru bisa nentuin gimana ngubah nya.

**Reply by animeblogs** [reply]
```
<video controls="" data-poster="image.jpg" id="player" playsinline="">
<source src="video.mp4" type="video/mp4" size="360">
<source src="video.mp4" type="video/mp4" size="480">
<source src="video.mp4" type="video/mp4" size="720">
<source src="video.mp4" type="video/mp4" size="1080">
</video>
```

**Reply by animeblogs** [reply]
pakai [plyr.io]([https://plyr.io/](https://plyr.io/))

**Comment by animeblogs** [comment]
itu template material igniel header gimana cara pasangnya di thema seleseven min?

**Reply by siwaluh** [reply]
harus dibuat dari awal template nya.

**Comment by bringbackold** [comment]
bang, mau tanya dong (tentang sitemap pagination) bisa gak sih itu background color manga, manhwa jadi beda? lalu ratingnya sesuai sama labelnya, jangan semuanya 5 bintang

**Reply by bringbackold** [reply]
sama warna statusnya juga bang siwaluh

**Reply by siwaluh** [reply]
1. Css Label Type
```
.cp1_type.Manhwa {
  background-color: red;
}

.cp1_type.Manhua {
  background-color: blue;
}

.cp1_type.Manga {
  background-color: yellow;
}
```
2. Css label status
```
.cp1_status {
 background-color: red;
}

.cp1_status.Ongoing {
 background-color: yellow;
}
```

**Reply by siwaluh** [reply]
Untuk rating sudah di perbaiki (update script component_html).
atau cari kode `/./g` ganti menjadi `/\./g`
